pages/common/shellcheck.md

```diff
@@ -1,20 +1,33 @@
 # shellcheck
 
 > Shell script static analysis tool.
-> More information: <https://www.shellcheck.net/>.
+> Check shell scripts for errors, usage of deprecated/insecure features, and bad practices.
+> More information: <https://www.shellcheck.net>.
 
 - Check a shell script:
 
-`shellcheck {{file.sh}}`
+`shellcheck {{path/to/script.sh}}`
 
-- Override script's shebang:
+- Check a shell script interpreting it as the specified shell dialect (overrides the shebang at the top of the script):
 
-`shellcheck --shell {{sh|bash|ksh}} {{file.sh}}`
+`shellcheck --shell {{sh|bash|dash|ksh}} {{path/to/file.sh}}`
```

Comment:
To keep consistent, should it be

```suggestion
`shellcheck --shell {{sh|bash|dash|ksh}} {{path/to/script.sh}}`
```

Reply:
Do you use `Hey dude, run this sh script`? I say `sh file` and `bash script` so I vote for `file.sh` :rofl:

Reply:
@einverne you're right—good catch.

You think we should use `file.sh`?  I'm fine with that—we should also change it in #5714, as well, then. @sbrl

Reply:
It really doesn't matter. I suggest using whichever is more clear on a case-by-case basis - this isn't something that needs standardising or worrying too much over consistency.